source/extensions/filters/http/peer_metadata/config.proto

```diff
@@ -37,6 +37,9 @@ message Config {
 
   // This method uses Istio HTTP metadata exchange headers, e.g. `x-envoy-peer-metadata`. Removes these headers if found.
   message IstioHeaders {
+    // Strip x-envoy-peer-metadata and x-envoy-peer-metadata-id headers on HTTP requests to services outside the mesh.
+    // Detects upstream clusters with `istio` and `external` filter metadata fields
```

Comment:
And based on PASSTHROUGH name it appears?

Reply:
I can either explicitly check for it here, or I can add the 
```
filter_metadata: 
  istio:
    external: true
```
to the passthrough cluster in the controlplane

Reply:
leaving logic in proxy for now